libs/common/src/sql-db-connection/migrations/1716990167763-AddPaymentTable.ts

```diff
@@ -0,0 +1,67 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddPaymentTable1716990167763 implements MigrationInterface {
+  name = 'AddPaymentTable1716990167763';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TABLE "Payment" (
+                "lastUpdatedAt" datetime2 NOT NULL CONSTRAINT "DF_3c7c91b94fc3c2cb358a90fa629" DEFAULT getdate(),
```

Comment:
I think we looked at putting these as the last columns instead of first columns before, I think francesca did this before but I'm not sure if it was a manual migration change or something else with the entities, @francescastocco do you remember?

Reply:
It was something Francesca did manually (#3027) - I'll make that fix now